### PR TITLE
Subtabs scrolling

### PR DESCRIPTION
**Pull request by dafortin**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

Note: content + actions are not scrolling.

# CHANGELOG
* **Fixed** subtabs scrolling

### CODE REVIEW

**Comment by sglanzer-fire** [comment]
I wish you wouldn't have merged this without a review - it's not complete imo

File: addon/styles/_frost-detail-subtab.scss [hunk 1]
```diff
@@ -4,9 +4,6 @@ $frost-detail-tab-inactive: $frost-color-grey-5;
 
 .frost-detail-subtab {
   position: relative;
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
```
Comment (sglanzer-fire) [inline addon/styles/_frost-detail-subtab.scss:9]: Why are you removing centering?

File: addon/styles/_frost-detail-subtabs.scss [hunk 2]
```diff
@@ -6,6 +6,9 @@ $frost-detail-tab-inactive: $frost-color-grey-5;
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 100px;
-  margin: 20px 0 0 20px;
+  width: 120px;
+  margin: 10px 0 ;
+  padding: 0 10px 0 20px;
+  overflow-y: scroll;
```
Comment (sglanzer-fire) [inline addon/styles/_frost-detail-subtabs.scss:12]: Don't use native scroll, use {{#frost-scroll}} or the scrollbar will appear differently to the rest of the content

File: addon/styles/_frost-detail-subtabs.scss [hunk 3]
```diff
@@ -6,6 +6,9 @@ $frost-detail-tab-inactive: $frost-color-grey-5;
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 100px;
-  margin: 20px 0 0 20px;
+  width: 120px;
+  margin: 10px 0 ;
+  padding: 0 10px 0 20px;
```
Comment (sglanzer-fire) [inline addon/styles/_frost-detail-subtabs.scss:11]: This doesn't appear equivalent to the previous spacing - the right padding and removal of 10px off the top margin

File: addon/styles/_frost-detail-tab-content.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 .frost-detail-tab-content {
   flex: 1;
-  margin: 10px;
+  margin: 10px 10px 10px 0;
```
Comment (sglanzer-fire) [inline addon/styles/_frost-detail-tab-content.scss:3]: Why is this being changed?

**Comment by rox163** [comment]
my apologies. I assumed it was good to go :(

File: addon/styles/_frost-detail-subtab.scss (hunk 1 above)
Comment (dafortin) [reply addon/styles/_frost-detail-subtab.scss:9]: This was causing issues. It was resizing the icons. Everything is still centered.

File: addon/styles/_frost-detail-subtabs.scss (hunk 2 above)
Comment (dafortin) [reply addon/styles/_frost-detail-subtabs.scss:12]: ok

File: addon/styles/_frost-detail-subtab.scss (hunk 1 above)
Comment (sglanzer-fire) [reply addon/styles/_frost-detail-subtab.scss:9]: What's responsible for centering it?

**Comment by sglanzer-fire** [comment]
All good, but we're on the home stretch and I'm going to be ultra-vigilant about things shifting :)

File: addon/styles/_frost-detail-subtab.scss (hunk 1 above)
Comment (dafortin) [reply addon/styles/_frost-detail-subtab.scss:9]: I will try with the `frost-scroll` and rework this.

**Pull request merged**